options/options.go

```diff
@@ -795,5 +803,240 @@ type EngineOptions struct {
 	Meta    map[string]interface{}
 }
 
+// ErrorsConfig extracted errors handling configuration.
+type ErrorsConfig struct {
+	Retry  map[string]*RetryConfig
+	Ignore map[string]*IgnoreConfig
+}
+
+// RetryConfig represents the configuration for retrying specific errors
```

Comment:
```suggestion
// RetryConfig represents the configuration for retrying specific errors.
```
strict-list complains about a comment without a dot at the end.